src/components/templates/Preplaya/Preplaya.tsx

```diff
@@ -151,8 +151,10 @@ const Preplaya = () => {
           }}
         >
           <div className="demo-message">
-            This is a demo of how camps and art will look on the final,
-            fully-interactive playa.
+            <span style={{ marginLeft: "10px" }}>
```

Comment:
i would keep this text centred and increase the padding so that the tab doesn't overlap it